xtos/include/sof/lib/dma.h

```diff
@@ -532,6 +532,15 @@ int dma_buffer_copy_from(struct comp_buffer __sparse_cache *source,
 			 struct comp_buffer __sparse_cache *sink,
 			 dma_process_func process, uint32_t source_bytes);
 
+/*
+ * Used when copying DMA buffer bytes into multiple sink buffers, one at a time using the provided
+ * conversion function. DMA buffer consume should be performed after the data has been copied
+ * to all sinks.
+ */
```

Comment:
don't think we need this same comment in both the header and the source - when documenting with Doxygen, I think we only comment in the header? But please check, not sure